CONTRIBUTING.md

```diff
@@ -94,7 +94,7 @@ NuGet members may contribute directly to the main remote.
 1. Run dotnet code formatters and correct any errors.
     * You can use `Format Document` in VS: 
 
-       `Ctrl+K, Ctrl+D` or Edit > Advanced > Format Document (https://docs.microsoft.com/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2019#text-editor)
+       `Ctrl+K, Ctrl+D` or Edit > Advanced > Format Document (https://learn.microsoft.com/en-us/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2022#bkmk_text-editor-context-specific-shortcuts)
```

Comment:
Should we have `en-us` in the URL?  Or should we let learn.microsoft.com redirect the customer to their preferred language automatically, like we're required to when linking to other docs pages in our own official docs?